template/javascript/base/eslint.config.js

```diff
@@ -13,11 +16,18 @@ export default [
   },
 
   js.configs.recommended,
-  ...pluginVue.configs['flat/recommended'],
+  ...vue.configs['flat/recommended'],
+  ...vuetify.configs['flat/base'],
```

Comment:
Doesn't have to be part of this P.R., but would love to include vuetify eslint config here.

Reply:
As a follow-up, the vuetify eslint plugin is really only for upgrading projects. It doesn't make much sense for new ones.